### PR TITLE
Mostly zero change cleanups and stylings

### PR DESCRIPTION
**Pull request by SzieberthAdam**

### CODE REVIEW

File: Assets/ActionInjectorHandler.cs [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 
 public class ActionInjectorHandler : MonoBehaviour
 {
     private ConcurrentQueue<Action> Queue;
 
-    void Start()
+
+
+    ///////////////////////////////////////////////////////////////////////////
+    //  MONOBEHAVIOUR METHODS  ////////////////////////////////////////////////
```
Comment (christerk) [inline Assets/ActionInjectorHandler.cs]: No, this is not how to do this.

If you're doing grouping of code like this, use `#region`

File: Assets/ActionInjectorHandler.cs [hunk 2]
```diff
@@ -35,10 +39,14 @@ void Update()
         }
     }
 
+
+
+    ///////////////////////////////////////////////////////////////////////////
+    //  CUSTOM METHODS  ///////////////////////////////////////////////////////
```
Comment (christerk) [inline Assets/ActionInjectorHandler.cs]: `#region`

**Comment by SzieberthAdam** [comment]
Mostly zero change code uniformization and cleanups. I looked over the whole codebase multiple times for this. I really think that readability got better and the code look nicer after these changes. Tested with the Unity Editor.

Exceptions (minor real changes):
- bc4bf88: Coach color uses settings
- 946dd92: Gender Nominative field got to private to match other fields
- 1280706: added a using of Model.Types
- abde896: Pass Block color uses settings

File: Assets/BlockDiceHandler.cs [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 using UnityEngine;
 using UnityEngine.UI;
-using Fumbbl.Model;
 using static Fumbbl.Model.Types.BlockDie;
 
 public class BlockDiceHandler : MonoBehaviour
 {
+    private ViewObjectList<BlockDie> BlockDice;
+    private static Vector2 FullSize = new Vector2(40, 40);
+    private static Vector2 SmallSize = new Vector2(30, 30);
```
Comment (christerk) [inline Assets/BlockDiceHandler.cs]: If you are going to organize fields and properties, public before private is the way to go.

File: Assets/BlockDiceHandler.cs [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
 
-    private ViewObjectList<BlockDie> BlockDice;
 
-    public Transform ContentObject;
-
-    private static Vector2 FullSize = new Vector2(40, 40);
-    private static Vector2 SmallSize = new Vector2(30, 30);
-    private static Vector2 SpacerSize = new Vector2(12, 30);
+    ///////////////////////////////////////////////////////////////////////////
+    //  MONOBEHAVIOUR METHODS  ////////////////////////////////////////////////
```
Comment (christerk) [inline Assets/BlockDiceHandler.cs]: `#region`. Not adding further comments about this particular style.

File: Assets/FieldHandler.cs [hunk 5]
```diff
@@ -1,48 +1,70 @@
...
 using Fumbbl.View;
-using System;
-using System.Collections.Generic;
 using System.Linq;
 using UnityEngine;
-using UnityEngine.UI;
 
 public class FieldHandler : MonoBehaviour
 {
+    private GameObject Ball;
```
Comment (christerk) [inline Assets/FieldHandler.cs]: public before private

File: Assets/FieldHandler.cs [hunk 6]
```diff
@@ -1,48 +1,70 @@
...
-    public TMPro.TextMeshProUGUI AwayTeamText;
+    ///////////////////////////////////////////////////////////////////////////
+    //  MONOBEHAVIOUR METHODS  ////////////////////////////////////////////////
+    ///////////////////////////////////////////////////////////////////////////
 
-    private GameObject Ball;
-    private ViewObjectList<Player> Players;
-    private ViewObjectList<PushbackSquare> PushbackSquares;
-    private ViewObjectList<TrackNumber> TrackNumbers;
+    private void OnDestroy()
```
Comment (christerk) [inline Assets/FieldHandler.cs]: I want life cycle methods grouped together. Start() followed by OnDestroy() to make it easier to follow when you read the code.

File: Assets/Scripts/Ffb/Dto/ModelChanges/FieldModelAddPushbackSquare.cs [hunk 7]
```diff
@@ -4,6 +4,8 @@ public class FieldModelAddPushbackSquare : ModelChange
     {
         public PushbackSquare modelChangeValue;
 
-        public FieldModelAddPushbackSquare() : base("fieldModelAddPushbackSquare") { }
+        public FieldModelAddPushbackSquare() : base("fieldModelAddPushbackSquare")
```
Comment (christerk) [inline Assets/Scripts/Ffb/Dto/ModelChanges/FieldModelAddPushbackSquare.cs]: This is unnecessary. The constructor calls the base class constructor and doesn't do anything and won't ever do anything beyond that. Leave it as it was.

File: Assets/Scripts/Model/Types/Player.cs [hunk 8]
```diff
@@ -5,28 +5,47 @@ namespace Fumbbl.Model.Types
 {
     public class Player : ViewObject<Player>
     {
-        public string Id;
-        public string PositionId;
-        public string Name;
-        public string PortraitURL;
-        public Types.Coordinate Coordinate { get; set; }
+        public Player()
```
Comment (christerk) [inline Assets/Scripts/Model/Types/Player.cs]: As a general rule, fields and properties before constructor. I know the DTOs have the constructor first and if you insist on unifying this, it's better to change the DTOs to match the more general case.

File: Assets/SettingsHandler.cs [hunk 9]
```diff
@@ -83,46 +112,43 @@ public void ShowPanel(GameObject panel)
...
 
-    public void SwitchToPreviousScene()
+    private int GetResolutionKey(int width, int height)
     {
-        MainHandler.Instance.SetScene(FFB.Instance.PreviousScene);
+        return width * 100000 + height;
     }
 
-    public void Quit()
+    public void UpdateAbstractIcons()
```
Comment (christerk) [inline Assets/SettingsHandler.cs:96]: public before private if you're reorganizing methods.

File: Assets/GameBrowserEntry.cs [hunk 10]
```diff
@@ -1,24 +1,41 @@
...
+    {
+        FFB.Instance.Stop();
+        FFB.Instance.Connect(matchDetails.id);
+        MainHandler.Instance.SetScene(MainHandler.SceneType.ConnectScene);
+    }
+
+
+
+    ///////////////////////////////////////////////////////////////////////////
+    //  CUSTOM METHODS  ///////////////////////////////////////////////////////
```
Comment (christerk) [inline Assets/GameBrowserEntry.cs]: Don't need to state something as generic as "custom methods". Use `#region` to group monobehaviour and leave these outside that region.

**Review by christerk — CHANGES_REQUESTED** [review]
This is a way too big PR. If you're doing this type of style cleanup, do separate PRs for each type of change to keep things small.

Also, do not have actual code changes in a PR like this as it becomes impossible to track those when looking back at commit history.

**Comment by SzieberthAdam** [comment]
I give further comments to the individual commits to help reviewing:
- 7adbd14: Zero change. Methods sorted alphabetically.
- 483f9b1: Zero change. Only removed trailing ` {}`s. 
- 593ca1b: Zero change. Added an empty line.
- bc4bf88: **Minor change**: Coach color uses settings.
- 946dd92: **Minor change**: Gender Nominative field got to private to match other fields.
- 6e63d63: Zero change. Fields ordered alphabetically.
- 99dc591: Zero change. I expected MonoBehaviour subclasses' got their methods and fields just throwed in. Made some order and visually separated the custom and the mono methods.
- 7c87d5c: **Minor change**: Some access modes were explicit some were excluded. I made all of them explicit here.
- 360dc4f **Minor change**: The code were full of unnecessary using directives. I removed them where the IDE indicated it.
- e607c5c: Zero change. The ModelChange class constructors had their curly brackets put differently. Some were at the end like ` {}` and some were put into the subsequent two lines. I made all to the latter style.
- b7b45ed **Minor change**. Using directives again. Missed to click to stage them when I commited 360dc4f,
- fc382ea Zero change: Add and remove empty lines.
- 1280706 **Minor change**: added a using of Model.Types and chenged the qualified name of Coordinates
- 0cdd824 Zero change. Moved the constructor up to match other modules approach.
- b36def2 Zero change. Ordered the fields.
- abde896 **Minor change**: Pass Block Log text uses settings color
- ae35902 Zero change: Various. Missed to save them in the editor in the first place.

**Comment by SzieberthAdam** [comment]
Number of changed files has been grown as I moved the constructors of the DTOs after the fields.

**Comment by christerk** [comment]
Resolve the conflict please.

**Comment by SzieberthAdam** [comment]
Resolved. LiginHandler.Login() got public in the meantime to be able to get called by the login scene button.

**Pull request merged**